decidim-proposals/lib/decidim/content_renderers/proposal_renderer.rb

```diff
@@ -23,8 +23,8 @@ def render
             proposal = GlobalID::Locator.locate(proposal_gid)
             Decidim::Proposals::ProposalPresenter.new(proposal).display_mention
           rescue ActiveRecord::RecordNotFound
-            proposal_id = proposal_gid.split("/").last
-            "<#{I18n.t("decidim.content_renderers.proposal_renderer.proposal_not_found", proposal_id: proposal_id)}>"
+            Rails.logger.warn("ProposalRenderer can not render unexisting proposal with gid: #{proposal_gid}")
+            "<RecordNotFound #{proposal_gid}>"
```

Comment:
Can we render the format we are parsing for? Eg, if we look for `#123` to render IDs, then try to re-render `#123` if the proposal cannot be found

Reply:
I can but then it wont be able to distinguish it from a hashtag, I could use the ~123 notation which is also recognized by the corresponding parser.

Reply:
`~123` looks weird to me, but maybe I'm too used to how GitHub works 😂 

But yes, that was why I tried to say! 😄